src/sdc11073/mdib/msgreader.py

```diff
@@ -1,6 +1,6 @@
 from lxml import etree as etree_
 import copy
-from .. import namespaces
+from .. import namespaces, xmlparsing
```

Comment:
I would prefer it to have no relative imports.

Reply:
done